src/rust/bitbox02/src/memory.rs

```diff
@@ -29,6 +30,21 @@ pub fn set_device_name(name: &str) -> Result<(), ()> {
     }
 }
 
+#[cfg(feature = "testing")]
+pub mod testing {
+    extern crate alloc;
+    pub static mut SET_DEVICE_NAME_EXPECTED_NAME: Option<alloc::string::String> = None;
+    pub static mut SET_DEVICE_NAME_RESULT: Result<(), ()> = Ok(());
+}
+
+#[cfg(feature = "testing")]
```

Comment:
the whole thing could be split into two files, memory.rs and memory_test.rs

Reply:
:+1: